.changeset/ninety-panthers-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-button": patch
+---
+
+[Changed] - Test change please ignore
```

Comment:
Guessing you'll be removing this before merging 😂

Reply:
Thanks for the reminder 😄